website/docs/api/doc.mdx

```diff
@@ -752,22 +752,23 @@ The L2 norm of the document's vector representation.
+| `text`                                        | A string representation of the document text. ~~str~~                                                                                          |
+| `text_with_ws`                                | An alias of `Doc.text`, provided for duck-type compatibility with `Span` and `Token`. ~~str~~                                                  |
+| `mem`                                         | The document's local memory heap, for all C data it owns. ~~cymem.Pool~~                                                                       |
+| `vocab`                                       | The store of lexical types. ~~Vocab~~                                                                                                          |
+| `tensor`                                      | Container for dense vector representations. ~~numpy.ndarray~~                                                                                  |
+| `user_data`                                   | A generic storage area, for user custom data. ~~Dict[str, Any]~~                                                                               |
+| `lang`                                        | Language of the document's vocabulary. ~~int~~                                                                                                 |
+| `lang_`                                       | Language of the document's vocabulary. ~~str~~                                                                                                 |
+| `user_hooks`                                  | A dictionary that allows customization of the `Doc`'s properties. ~~Dict[str, Callable]~~                                                      |
+| `user_token_hooks`                            | A dictionary that allows customization of properties of `Token` children. ~~Dict[str, Callable]~~                                              |
+| `user_span_hooks`                             | A dictionary that allows customization of properties of `Span` children. ~~Dict[str, Callable]~~                                               |
+| `has_unknown_spaces`                          | Whether the document was constructed without known spacing between tokens (typically when created from gold tokenization). ~~bool~~            |
+| `_`                                           | User space for adding custom [attribute extensions](/usage/processing-pipelines#custom-components-attributes). ~~Underscore~~                  |
+| `activations` <Tag variant="new">4.0</Tag>    | A dictionary of activations per trainable pipe (available when the `save_activations` option of a pipe is enabled). ~~Dict[str, Option[Any]]~~ |
+| `ents_spangroup` <Tag variant="new">4.0</Tag> | All entitity `Span` instances (as stored in `.ents`) as [`SpanGroup`](/api/spangroup). ~~SpanGroup~~                                           |
```

Comment:
This isn't a `Doc` attribute, it should be added as a property similar to `Doc.ents`.